### PR TITLE
Feature: Add slack instructions

### PR DESCRIPTION
**Pull request by craiglonsdale**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,8 +12,9 @@ npm install console-squeal
 
 ```js
 var squeal = require('console-squeal');
-squeal.sentry_dsn = 'my_sentry_dsn';
-squeal.createLoggers(['console', 'file', 'sentry']).start();
+squeal.sentryDsn = 'my_sentry_dsn';
+squeal.slackOptions.webhook = 'my_slack_webhook';
+squeal.createLoggers(['console', 'file', 'sentry', 'slack']).start();
```
Comment (programbo) [inline README.md]: Where does this code go?

File: README.md [hunk 2]
```diff
@@ -12,8 +12,9 @@ npm install console-squeal
...
 ```
 
 All subsequent calls to console.log/debug/info/warm/error will be handled by the Winston
```
Comment (jedimonkey) [inline README.md]: typo: warm: warn.

Comment (craiglonsdale) [inline README.md]: naw it's warm ;)

File: README.md (hunk 1 above)
Comment (craiglonsdale) [inline README.md]: Doesn't matter, wherever you want I guess

Comment (jedimonkey) [inline README.md]: I guess it's worth noting it should go in the init process - as early as possible?

**Pull request merged**